crates/rolldown/src/chunk/mod.rs

```diff
@@ -79,7 +77,7 @@ impl Chunk {
     options: &SharedOptions,
     graph: &LinkStageOutput,
     chunk_graph: &ChunkGraph,
-  ) -> BatchedResult<ChunkRenderReturn> {
+  ) -> Result<ChunkRenderReturn> {
```

Comment:
```suggestion
  ) -> BatchedResult <ChunkRenderReturn> {
```

Not in the edge of the program, should keep `BatchedResult`.